airflow/utils/log/logging_mixin.py

```diff
@@ -58,6 +58,20 @@ def _set_context(self, context):
             set_context(self.log, context)
 
 
+class RemoteLoggingMixin:
```

Comment:
```suggestion
class ExternalLoggingMixin:
```
What do you think to change the name? Remote logging is an existing mechanism that allows access to logs from Web UI. In my opinion, external logs better describe this feature.

Reply:
Sounds good, I'll change it.